packages/j/jabref/package.yml

```diff
@@ -1,34 +1,36 @@
 name       : jabref
-version    : '5.5'
-release    : 4
+version    : '5.15'
+release    : 5
 source     :
-    - https://github.com/JabRef/jabref/archive/refs/tags/v5.5.tar.gz : af0d7c5566aec94ffa8b5912c116956ccf17331e5b46bb5adbd6afeaa85ac230
+    - https://github.com/JabRef/jabref/archive/refs/tags/v5.15.tar.gz : 2bf75d7f96aa1f5c74b675e3e0b4dda703027b3c3d8ede0616dd2218f5a74259
+homepage   : https://jabref.org
 license    : MIT
 component  : office
-summary    : Graphical Java application for managing BibTeX and BibLaTeX (.bib) databases
+summary    : JabRef Bibliography Management 
 description: |
-    Graphical Java application for managing BibTeX and BibLaTeX (.bib) databases
+    JabRef is an open-source, cross-platform citation and reference management tool.
+    JabRef helps you to collect and organize sources, find the paper you need and discover the latest research.
 builddeps  :
+    - binutils
     - gradle
-    - openjdk-17
+    - openjdk-21
 rundeps    :
-    - openjdk-17
+    - openjdk-21
 networking : yes
 environment: |
-    export JAVA_HOME=/usr/lib64/openjdk-17
+    export JAVA_HOME=/usr/lib64/openjdk-21
     export PATH=$JAVA_HOME/bin:$PATH
 
     export GRADLE_USER_HOME=$workdir
 build      : |
-    gradle \
-      --no-daemon \
-      -PprojVersionInfo=$version \
-      -PprojVersion=$version \
-      assemble
+    ./gradlew assemble --no-daemon --no-build-cache -PprojVersionInfo=$version -PprojVersion=$version 
+    ./gradlew jlink
```

Comment:
Is there a reason for using the bundled gradle instead of the system version?

Reply:
Good catch.  Thank you.
Originally this would not build and I was trying lots of things and for some reason using their wrapper worked.  This is also what was documented on their website.   
At any rate - I did mean to revert it and try it with the previous build commands but I neglected to do that.
I just tried to rebuild it with the system gradle and it still builds.

I will make those changes and update the PR.    
Thank you for checking things so closely to ensure we have nice tight and consistent builds

Reply:
I have updated the build to use the system version.